File: week6/pset6/mario/more/mario.py
from cs50 import get_int

# get height from user - if input is less than 1 and  or more than 8, repeat
while True:
    height = get_int("Height (between 1 and 8): ")
    if height in range(1, 9):
        break

# print first pyramid
for i in range(0, height):
    for j in range(0, height):
        if ((i + j) >= height - 1):
            print("#", end="")
        else:
            print(" ", end="")

    # print space between the pyramids
    print("  ", end="")

    # print the second pyramid
    for k in range(0, height):
        if ((k == i) or (i > k)):
            print("#", end="")
    print("")

File: week01/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Heigh: ");
    }
    while (height > 8 || height < 1);

    // logic first pyramid
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height; j++)
        {
            // if i + j is greater or equals to height - 1 so print hash, otherwise print space
            if ((i + j) >= height - 1)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        // spaces between pyarmid
        printf("  ");


        // second pyramid without space logic
        for (int k = 0; k < height; k++)
        {
            if ((k == i) || (i > k))
            {
                printf("#");
            }
        }
        printf("\n");
    }
}